[2025-10-21T17:49:32.539+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-21T17:49:32.581+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: airbnb_etl_pipeline.transform_airbnb_listings manual__2025-10-21T17:45:45.179461+00:00 [queued]>
[2025-10-21T17:49:32.592+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: airbnb_etl_pipeline.transform_airbnb_listings manual__2025-10-21T17:45:45.179461+00:00 [queued]>
[2025-10-21T17:49:32.594+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-10-21T17:49:32.608+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transform_airbnb_listings> on 2025-10-21 17:45:45.179461+00:00
[2025-10-21T17:49:32.616+0000] {standard_task_runner.py:63} INFO - Started process 266 to run task
[2025-10-21T17:49:32.620+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'airbnb_etl_pipeline', 'transform_airbnb_listings', 'manual__2025-10-21T17:45:45.179461+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/airbnb_etl_dag.py', '--cfg-path', '/tmp/tmpbbli1vlv']
[2025-10-21T17:49:32.625+0000] {standard_task_runner.py:91} INFO - Job 16: Subtask transform_airbnb_listings
[2025-10-21T17:49:32.692+0000] {task_command.py:426} INFO - Running <TaskInstance: airbnb_etl_pipeline.transform_airbnb_listings manual__2025-10-21T17:45:45.179461+00:00 [running]> on host 7b5c622c061b
[2025-10-21T17:49:32.784+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='airbnb_etl_pipeline' AIRFLOW_CTX_TASK_ID='transform_airbnb_listings' AIRFLOW_CTX_EXECUTION_DATE='2025-10-21T17:45:45.179461+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-21T17:45:45.179461+00:00'
[2025-10-21T17:49:32.785+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-21T17:49:32.799+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-21T17:49:32.800+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/scripts/etl_script.py", line 575, in transform1
    airdna_data = pd.json_normalize(all_listings)
                                    ^^^^^^^^^^^^
NameError: name 'all_listings' is not defined
[2025-10-21T17:49:32.826+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=airbnb_etl_pipeline, task_id=transform_airbnb_listings, run_id=manual__2025-10-21T17:45:45.179461+00:00, execution_date=20251021T174545, start_date=20251021T174932, end_date=20251021T174932
[2025-10-21T17:49:32.837+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 16 for task transform_airbnb_listings (name 'all_listings' is not defined; 266)
[2025-10-21T17:49:32.877+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-10-21T17:49:32.899+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-21T17:49:32.902+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
